multiworld/envs/mujoco/sawyer_xyz/sawyer_reach.py

```diff
@@ -126,9 +125,9 @@ def _reset_hand(self):
             self.data.set_mocap_quat('mocap', np.array([1, 0, 1, 0]))
             self.do_simulation(None, self.frame_skip)
 
-    def reset(self, resample_on_reset=True):
+    def reset(self):
```

Comment:
At this point, the code is the same as the base class, so feel free to delete this.

Reply:
by this you mean delete the reset function?